ironfish-cli/src/commands/logs.ts

```diff
@@ -23,11 +23,19 @@ export default class LogsCommand extends IronfishCommand {
     const response = this.sdk.client.getLogStream()
 
     for await (const value of response.contentStream()) {
+      let parsedArgs
+      try {
+        parsedArgs = IJSON.parse(value.args) as unknown[]
+      } catch {
+        this.logger.error(`Failed to deserialize args: ${value.args}`)
+        continue
+      }
```

Comment:
Should we let this crash, or render the error? So users can send it to us? These will be things we need to fix, right?

I'd like to either let it crash, or just use `ErrorUtils.renderError(error, true)`

Reply:
I'll re-throw the error here so it crashes